app/__init__.py

```diff
@@ -38,6 +47,11 @@ def create_app() -> Flask:
     toolbar.init_app(app)
     notification_service.init_app(app)
     talisman.init_app(app, **app.config["TALISMAN_SETTINGS"])
+    login_manager.init_app(app)
+
+    @login_manager.user_loader  # type: ignore[misc]
+    def load_user(user_id: str) -> User | None:
+        return db.session.get(User, user_id)
```

Comment:
It's (almost definitely) a nit but could this call out to `interfaces.users` so that anything that can query the db sits in one place?

Reply:
oh - yeah - good catch